benchmarks/load-testing/wrk/query.sh

```
@@ -0,0 +1,35 @@
+sleep 5
+
+echo ""
+echo "---------------------------------------------------------"
+echo " Running Warmup $name"
+echo " wrk -H 'Host: $server_host' -H 'Accept: $accept' -H 'Connection: keep-alive' --latency -d $duration -c $max_concurrency --timeout 8 -t $max_threads \"${url}2\""
+echo "---------------------------------------------------------"
+echo ""
+wrk -H "Host: $server_host" -H "Accept: $accept" -H "Connection: keep-alive" --latency -d $duration -c $max_concurrency --timeout 8 -t $max_threads "${url}2"
+sleep 5
+
+for c in $levels
```

Shellcheck

[warning] levels is referenced but not assigned.

(SC2154)